docs/content/latest/api/ysql/commands/ddl_create_table.md

```diff
@@ -50,6 +53,8 @@ table_constraint ::= [ CONSTRAINT constraint_name ]
                      { CHECK ( expression ) |
                        PRIMARY KEY ( column_names ) |
                        FOREIGN KEY ( column_names ) references_clause }
+
+storage_param ::= storage_parameter [ = value ]
```

Comment:
`storage_parameter` should be something like `name` -- then `storage_param` could be renamed to `storage_parameter`